pkg/sigv4/tripper_test.go

```diff
@@ -0,0 +1,35 @@
+package sigv4_test
+
+import (
+	"github.com/grafana/xk6-output-prometheus-remote/pkg/sigv4"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestTripper_request_includes_required_headers(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we make this test `t.Parallel()` ?

Reply:
addressed in [f3e0598](https://github.com/grafana/xk6-output-prometheus-remote/pull/169/commits/f3e0598fb8897a0600f8f5030b8696c25e1553d0)